### PR TITLE
made README more readable; fixed some errors

### PR DESCRIPTION
**Pull request by sreenikethMadgula**
This rectifies some minor errors in the README file. 
The comments in the commit explain the purpose of the changes made. To look at them, feel free to open the commit 911f77a

The changes bring about minor yet meaningful effect to the readability of the README file. Since this is the first file that any contributor would first have a look at, rectifying the errors in it is vital.

Please feel free to correct any changes; advise on adding new ones.
Thank you

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,23 +47,21 @@ Working on the indexing pipeline effectively requires internal access to the
 Internet Archive cluster and services, though some contributions and bugfixes
 are probably possible without staff access.
 
-To install dependencies for the first time, then run the tests (to ensure
-everything is working):
```
Comment (bnewbold) [inline README.md:51]: I think these indented lines ("make dep" and "make test") need to have empty lines before and after, so they end up as indented code blocks.

File: README.md [hunk 2]
```diff
@@ -47,23 +47,21 @@ Working on the indexing pipeline effectively requires internal access to the
...
-
-If developing the web interface, you will almost certainly need an example
+While developing the web interface, you will almost certainly need an example
 database running locally. A docker-compose file in `extra/docker/` can be used
 to run Elasticsearch 7.x locally. The `make dev-index` command will reset the
 local index with the correct schema mapping, and index any intermediate files
 in the `./data/` directory. We don't have an out-of-the-box solution for non-IA
 staff at this step (yet).
 
-After making changes to any user interface strings, the interface translation
```
Comment (bnewbold) [inline README.md:63]: I'm not sure what the intent was with this change, but I think it only removed a period. If the intent was to have the "make" command show as separate code, it needs to have 4 spaces in front of it, and be separated from the text paragraph with blank lines. In this case, I think having the command in-line is appropriate though.

**Review by bnewbold — CHANGES_REQUESTED** [review]
Thanks for your contribution! Most of these look like good improvements, but I made requested changes to two of them.

If you prefer, I can merge the changes you have made as-is, then make small tweaks after, or you can update the changes and I merge them directly.

File: README.md (hunk 1 above)
Comment (sreenikethMadgula) [reply README.md:51]: I understand. That actually was my intent, to get them to appear indented. I will correct this instantly.
Thank you

File: README.md (hunk 2 above)
Comment (sreenikethMadgula) [reply README.md:63]: Yeah, the objective was to get the `make` command to appear in a new line. I concur to having it inline; will change it right away

**Comment by sreenikethMadgula** [comment]
@bnewbold I've made the required changes in commit 925ad1. 
Please let me know if it looks alright, and if any further changes are to be made.
Thank you

**Review by bnewbold — APPROVED** [review]

**Pull request merged**